tasks/eth/006-MCP-L1/SignFromJson.s.sol

```diff
@@ -0,0 +1,346 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.15;
+
+import {SignFromJson as OriginalSignFromJson} from "../../../script/SignFromJson.s.sol";
```

Comment:
small nit: no need to give relative path, can always do a path relative to project root, ie `src/script/SignFromJson.s.sol". This makes it easier to refactor things, no need to use `../...` and change the import path during refactor

Reply:
+1 for absolute paths